packages/core/src/IconContainer/IconContainer.stories.tsx

```diff
@@ -0,0 +1,23 @@
+import { faAtom } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { Meta, StoryObj } from "@storybook/react";
+import { HvIconContainer } from "@hitachivantara/uikit-react-icons";
+
+const meta: Meta<typeof HvIconContainer> = {
+  title: "Components/Icon Container",
+  component: HvIconContainer,
+};
+export default meta;
+
+export const Main: StoryObj<typeof HvIconContainer> = {
+  parameters: {
```

Comment:
Let's remove `parameters` here: we are already testing this component visually inside `docs/foundation/Icons/Icons.stories.tsx` 🙏